austation/code/modules/mining/equipment/marker_beacons.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/stack/marker_beacon/attack_self(mob/user)
+	if(GLOB.total_beacons >= GLOB.max_beacons) //check that  there are not 100 in the world already
+		to_chat(user, "<span class='warning'>You can not place another beacon!</span>")
+		return
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

Reply:
. basically means "the return value of this proc", so when you're doing a parent set call at the end of the proc, it's a bit cleaner to do this :)